mlflow/models/python_api.py

```diff
@@ -142,6 +143,13 @@ def predict(
                 the logged model's dependency using `mlflow.models.update_model_requirements` API
                 without re-logging it. Note that a registered model is immutable, so you need to
                 register a new model version with the updated model.
+        extra_envs: If specified, a dictionary of extra environment variables will be passed to the
```

Comment:
Can we update the code block example below to show usage of this additional parameter? (We should also update the docs to show an example of this for a GenAI use case with some form of API KEY in a followup)